pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -421,6 +421,17 @@ type awsElasticBlockStoreProvisioner struct {
 var _ volume.Provisioner = &awsElasticBlockStoreProvisioner{}
 
 func (c *awsElasticBlockStoreProvisioner) Provision() (*v1.PersistentVolume, error) {
+	var fsType string = "ext4"
```

Comment:
I'd vote for skipping this for now and properly implement default fsType handling when this is implemented: https://github.com/kubernetes/kubernetes/issues/37804
The hard coded values were already there and thus this PR at least does not make the situation worse.